src/cmd/run.rs

```diff
@@ -203,6 +203,19 @@ async fn poll_integration<D: Database>(
     db: &D,
     integration: &Integration,
     sender: &Queue<ScanGitVCSReference>,
+) -> Result<(), Error> {
+    loop {
+        if let Err(err) = execute_poll_integration(db, integration, sender).await {
+            warn!("attempt to poll integration failed: {err:#?}");
```

Comment:
In technical writing generally, but especially logging (and error messages, but that's a different topic), prefer terseness.
Users already know the program is attempting something, just log the issue.

```
warn!("poll integration failed: {err:#?}");
```

Additionally, remember that log statements like this are intended for users to consume; prefer to format them using standard rules for English prose (just more terse). In this particular case, this just means capitalizing the first letter:
```
warn!("Poll integration failed: {err:#?}");
```

Finally, since this is a warning, make it clear why the user is even being alerted in the first place. Specifically, why is this a warning? What are the consequences of ignoring it? The context here is that users of any system will ignore warnings by default, and some of them will actually read the warnings. Respect the time of those that do by letting them know how to interpret the warning:
```suggestion
            warn!("Unable to poll '{integration}'. This may be temporary or due to a configuration issue. Details: {err:#?}");
```

This is also why we prefer to be terse: the goal is to make the message very information dense (so the few that read the message can understand the problem, ideally without having to look anywhere else) without being overly burdensome to read (because already the default is to ignore the message). These goals conflict, of course, so the exact tradeoffs are more of an art than a science- feel free to substitute wording you think is better!

Please apply this feedback to all similar instances.